refs/biblio.json

```diff
@@ -3343,6 +3343,15 @@
                 "deliveredBy": [
                     "http://data.europa.eu/w21/ac376c94-74cf-4dd7-ade7-267d6a4ec4dc"
                 ]
+            },
+            "20190528": {
+                "href": "https://joinup.ec.europa.eu/release/dcat-ap/121",
+                "title": "DCAT Application Profile for data portals in Europe. Version 1.2.1",
+                "rawDate": "2019-05-28",
+                "publisher": "European Commission",
+                "deliveredBy": [
```

Comment:
FYI: you don't need to repeat all of the data that stays the same across versions (e.g. deliveredby, publisher, etc.

Reply:
Thanks, @tobie . Good to know.